nasa-gibs/worldview
# WV-3049: Dismiss DDV notifications with local storage

## Description

This PR updates DDV alert notifications for location and zoom level to be permanently dismissed when a user clicks the X which is tracked in local storage. 

## How To Test

1. `git checkout wv-3049`
2. `npm run watch`
3. [Load Worldview at this URL](http://localhost:3000/?v=-96.13688944457311,24.826947164861053,-67.57165069345734,41.953986500805456&l=HLS_False_Color_Urban_Sentinel(bandCombo=%7B%22r%22%3A%22B12%22;%22g%22%3A%22B11%22;%22b%22%3A%22B04%22%7D),Coastlines_15m,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-04-08-T12%3A01%3A07Z)
4. Wait for the zoom level alert to appear and then click the X to dismiss
5. Reload the page and verify that the alert does not appear. 
6. [Load Worldview at this URL](http://localhost:3000/?v=-84.22467987918361,35.85818946335679,-82.77791177853655,36.72563728641425&l=HLS_False_Color_Urban_Sentinel(bandCombo=%7B%22r%22%3A%22B12%22;%22g%22%3A%22B11%22;%22b%22%3A%22B04%22%7D),Coastlines_15m,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-04-08-T12%3A01%3A07Z)
7. Wait for the location alert to appear and then click the X to dismiss
8. Reload the page and verify that the alert does not appear again

If you want to test again, open the dev tools (chrome), go to the application, click on local storage and delete the keys that say `dismissedDDVLocationAlert` and `dismissedDDVZoomAlert` & reload page. 



@nasa-gibs/worldview
